Update How It Works: Certificate-Based Authentication (#1078)
gravitational/next: pages/how-it-works/certificate-based-authentication-ssh-kubernetes/index.mdx

hideWave: true
border: none
h1: Certificate-Based Authentication
title: Certificates-based authentication for SSH, RDP, Kubernetes, database, and machine-to-machine access
description: Learn how Teleport implements certificate-based authentication for SSH, RDP, Kubernetes, database, and machine-to-machine access.
headerTitle: Certificate-Based Authentication
headerDescription: Teleport automates Certificate Authority (CA) management and issues short-lived and scoped certificates for SSH, RDP, Kubernetes, and database authentication. 
layout: howItWorks
reviews:
 - awin2
---

import NextImage from "next/image";
import Feedback from "components/Feedback";
import Sidebar from "components/Sidebar";

## The Basics

> The certificate-based authentication is the most secure form of authentication for SSH, RDP, Kubernetes and database access. But historically, it has been the most complicated to set up. One of our design goals for Teleport has been to make certificates easy.

Reviewer: It would be nice to explain why, it looks like we removed the Pros of Certs below.

Author: Added a short line on "why" in the following commit.

hideWave: true
border: none
h1: Certificate-Based Authentication

> title: Certificates-based authentication for SSH, RDP, Kubernetes, database, and machine-to-machine access

Reviewer 2: Should we add "applications" as well?

> description: Learn how Teleport implements certificate-based authentication for SSH, RDP, Kubernetes, database, and machine-to-machine access.

Reviewer 2: ```suggestion
description: Learn how Teleport implements certificate-based authentication for SSH, RDP, Kubernetes, databases, and machine-to-machine access.
```

headerTitle: Certificate-Based Authentication
headerDescription: Teleport automates Certificate Authority (CA) management and issues short-lived and scoped certificates for SSH, RDP, Kubernetes, and database authentication. 
layout: howItWorks
reviews:
 - awin2
---

import NextImage from "next/image";
import Feedback from "components/Feedback";
import Sidebar from "components/Sidebar";

## The Basics

> The certificate-based authentication is the most secure of them all, but historically, it has been the most complicated to manage the Certificate Authority (CA) and distribute server and client certificates, impeding its adoption at scale. One of our design goals for Teleport has been to securely automate CA and certificate issuance, making certificate-based authentication easy.

Reviewer 2: ```suggestion
Certificate-based authentication is the most secure form of authentication, but historically, it has been the most complicated to manage.  You have to run a Certificate Authority (CA) and distribute server and client certificates which impedes its adoption at scale. One of our design goals for Teleport has been to securely automate CA and certificate issuance, making certificate-based authentication easy.
```
